CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## 2.0.0
+
+Put component classes as files in the new `components` directory.\
+Change the instantiation of a component in the `get_component_object` method of the `Builder` class.
+
+**Warning**:
+
+This changes the `import` declaration (path) of how components should be imported.
```

Comment:
@sjamaan Can you please advice here?

Does the library still need to support the "Old style imports" ?
See the differences below (also in this CHANGELOG version 2.0.0).
Because I don't want to cause backwards compatibility breakage.

It's easy to also support "Old style imports" by putting those into `formiodata/components/__init__.py`
See this commit, where those imports are still commented (disabled).
Or does this lead to confusion about the import declaration/path, which should be avoided?

Additionally, is the file `formiodata/components/__init__.py` then still needed, adhering to the Python principles?

Reply:
> @sjamaan Can you please advice here?
> 
> Does the library still need to support the "Old style imports" ? See the differences below (also in this CHANGELOG version 2.0.0). Because I don't want to cause backwards compatibility breakage.

It's only breakage for those folks who are extending components, but as far as I know that's not really possible yet anyway, because custom component classes cannot currently be loaded due to the hardcoded component name -> class lookup that we currently have. Of course, the upcoming version will actually make that *possible*, so going forward in future versions you may need to be more careful with such changes.

> It's easy to also support "Old style imports" by putting those into `formiodata/components/__init__.py` See this commit, where those imports are still commented (disabled). Or does this lead to confusion about the import declaration/path, which should be avoided?

I think it's better to avoid, as it is *very* easy to forget adding new classes to the imports, and it creates confusion as to what the canonical way of importing a class would be.

> Additionally, is the file `formiodata/components/__init__.py` then still needed, adhering to the Python principles?

Yeah, if you omit it, I think Python cannot find the classes. It's a bit stupid to have empty `__init__.py` files, but yeah that's just how it is (AFAIK).